python/tools/create_inbox_message/create_inbox_message.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import ftrack_api
+
+session = ftrack_api.Session()
+
+recipients_user_names = ['<username>']
+message_content = 'Message body'
+
+# Query author and recipients. Note that the message will not appear if the
+# author and recipient is the same user.
+author = session.query(f"User where username is {session.api_user}").one()
+recipients = session.query(
+    "User where username in ('{}')".format("','".join(recipients_user_names))
```

Comment:
@seebergs as we are in py3 land, better use new f string formatting as you did in the previous line

Reply:
Good idea, added.